### PR TITLE
Bugfix/kill async processes

### PR DESCRIPTION
**Pull request by EduardKrieger**

### CODE REVIEW

File: runners/console/index.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import { Playbook } from "../../engine/playbook";
 import { ConsolePlatform, AsyncProcess } from "./consoleInterfaces";
 import * as path from 'path';
 import * as fs from "fs-extra";
-import * as psList from "ps-list";
 import { ConsoleUtils } from "./consoleUtils";
 
+const { listProcess } = require("process-list");
```
Comment (GuentherJulian) [inline runners/console/index.ts:11]: You need to add the package to the package.json file, don't you?
And the brackets are not needed I think.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "is-reachable": "^5.0.0",
     "mocha": "^8.2.1",
     "pegjs": "^0.10.0",
+    "process-list": "^2.0.0",
```
Comment (GuentherJulian) [inline package.json:17]: The other ps-list package is not needed anymore and can be removed

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/console/index.ts (hunk 1 above)
Comment (EduardKrieger) [reply runners/console/index.ts]: I have removed the brackets

File: package.json (hunk 2 above)
Comment (EduardKrieger) [reply package.json:17]: I have removed the unused package

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**